packages/manager/src/features/CloudPulse/Overview/GlobalFilters.tsx

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable no-console */
+// import { Dashboard, TimeDuration, TimeGranularity } from '@linode/api-v4';
+import { styled } from '@mui/material/styles';
+import Grid from '@mui/material/Unstable_Grid2';
+import * as React from 'react';
+
+import { WithStartAndEnd } from 'src/features/Longview/request.types';
+
+import { CloudPulseRegionSelect } from '../shared/RegionSelect';
+import { CloudPulseTimeRangeSelect } from '../shared/TimeRangeSelect';
+
+export interface GlobalFilterProperties {
+  handleAnyFilterChange(filters: FiltersObject): undefined | void;
+}
+
+export interface FiltersObject {
+  interval: string;
+  region: string;
+  resource: string[];
+  serviceType?: string;
+  timeRange: WithStartAndEnd;
+}
+
+export const GlobalFilters = React.memo((props: GlobalFilterProperties) => {
+  const [time, setTimeBox] = React.useState<WithStartAndEnd>({
+    end: 0,
+    start: 0,
+  });
+
+  const [selectedRegion, setRegion] = React.useState<string>();
+
+  const triggerGlobalFilterChange = () => {
+    const globalFilters = {} as FiltersObject;
```

Comment:
Better to avoid casting if possible.

```suggestion
    const globalFilters: FiltersObject = {};
```